src/praisonai/praisonai/kanban/sqlite_store.py

```
@@ -0,0 +1,565 @@
+            params.append(value)
+        
+        if not set_clauses:
+            return self._get_task_with_conn(task_id, conn)
+        
+        # Always update timestamp and version
+        set_clauses.append("updated_at = ?")
+        set_clauses.append("version = version + 1")
+        params.append(datetime.now(timezone.utc).isoformat())
+        params.append(task_id)
+        
+        query = f"UPDATE tasks SET {', '.join(set_clauses)} WHERE id = ?"
```

Ruff (0.15.14)

[error] Possible SQL injection vector through string-based query construction

(S608)